pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -249,6 +285,17 @@ var _ = Describe("ImageUpload", func() {
 			Expect(cmd()).NotTo(BeNil())
 		})
 
+		It("uploadProxyURL not configured", func() {
```

Comment:
Maybe rename the test to something like "should fail to upload if no uploadProxyURL is configured"

Reply:
it's part of ""Upload fails" context, we will have a duplicate title if I change the test name.